examples/01_plotting/plot_overlay.py

```diff
@@ -52,7 +52,7 @@
 
 
 ###############################################################################
-# Visualizing a probablistic atlas with plot_prob_atlas
+# Visualizing a probabilistic atlas with plot_prob_atlas
 # =====================================================
```

Comment:
```suggestion
# ======================================================
```

Reply:
Fixed, but there are such issues all over the place, to the point it feels no such rule is really enforced.